### ISSUE TITLE
The "items" key for an array causes attributes to go missing

### ISSUE THREAD

**Issue by benjaminoakes** [interagent/prmd#275]
Having, say, an array of strings documented as an attribute will cause that attribute to disappear from the output.  [This section](https://github.com/interagent/prmd/blob/6a84c6701ff8b65145665531292e97493f2fe806/lib/prmd/templates/schemata/helper.erb#L48) may be the culprit.

#### Input Schema

```json
{
  "$schema": "http://json-schema.org/draft-04/hyper-schema",
  "id": "/schemata/array_items_attribute_bug",

  "title": "Example",

  "type": "object",
  "strictProperties": true,

  "properties": {
    "urls": {
      "type": "array",
      "description": "This attribute will disappear from the documentation until the `items` property is removed.",
      "items": { "type": "string", "format": "uri" },
      "example": []
    }
  }
}
```

#### Output Markdown

```markdown
## <a name="resource-array_items_attribute_bug"></a>Example

### Attributes

| Name | Type | Description | Example |
| ------- | ------- | ------- | ------- |
```

(Notice that `urls` is missing.)

#### Input Schema

```json
{
  "$schema": "http://json-schema.org/draft-04/hyper-schema",
  "id": "/schemata/array_items_attribute_bug",

  "title": "Example",

  "type": "object",
  "strictProperties": true,

  "properties": {
    "urls": {
      "type": "array",
      "description": "This attribute will disappear from the documentation until the `items` property is removed.",
      "example": []
    }
  }
}
```

#### Output Markdown

```markdown
## <a name="resource-array_items_attribute_bug"></a>Example

### Attributes

| Name | Type | Description | Example |
| ------- | ------- | ------- | ------- |
| **urls** | *array* | This attribute will disappear from the documentation until the `items` property is removed. | `[]` |
```

(Notice that `urls` is present as it should have been in the previous example.)

## Details

ruby 2.3.0
prmd 0.8.0 (though this seems to be an issue in 0.11 as well)

**Comment by benjaminoakes** [comment]
This was fixed sometime between v0.8.0 and v0.10.0.  (Curiously, I don't see a tag for v0.9.0.)  There was another bug that was preventing us from upgrading past v0.8.0 -- which I'm going to be looking into next.  :smile:

**Comment by geemus** [comment]
Sorry I didn't get back to you sooner, but glad to hear you figured out a fix. Definitely let me know on the other blocker when you get a chance and I can try to help there as well.
